functions/src/judge.ts

```diff
@@ -61,7 +61,7 @@ export async function submit(req: Request, res: Response) {
   // TODO: Add the submission IDs to the user's data
   // TODO: For problems in the database, grab the data and use that for input/output
 
-  // C = 50, C++ = 54, Java = 62, Python: 71 (does not use pypy in default judge0)
+  // C = 50, C++ = 54, Java = 62, Python:c 71 (does not use pypy in default judge0)
```

Comment:
What is this `c`? Why does Python have a `:` when the others have `=`? Also, just curious, where do these numbers for the languages come from?

Reply:
Its their ids in judge0, just for references

Reply:
c is a typo